tests/infrastructure/instance_types/utils.py

```diff
@@ -43,3 +45,19 @@ def assert_instance_revision_and_memory_update(
     assert guest_memory == updated_memory, (
         "The Guest Memory in VMI is {guest_memory}, not updated to {updated_memory} after editing"
     )
+
+
+def assert_dmesg(vm: VirtualMachine, dmesg_pattern: str, expected_status: str) -> None:
+    assert run_ssh_commands(
+        host=vm.ssh_exec, commands=shlex.split(f"sudo dmesg | grep -iE '{dmesg_pattern}.*{expected_status}'")
+    )[0], f"Expected pattern '{dmesg_pattern}' with status '{expected_status}' not found in dmesg logs."
```

Comment:
since you call this function only once, no need to pass arguments dmesg_pattern and expected_status

one suggestion - how about making this function similar to other ones: save `dmesg | grep -i secureboot` to variable `output` and do `assert "enabled" in output`